### PR TITLE
feat: add github to colorstack socials 🐱

### LINKED ISSUE #37: Add GitHub as social link in the Member Profile home page 🏡
### Description

We display our ColorStack socials in the home page of the Member Profile, as such:

Screenshot 2024-03-29 at 8 44 03 AM

Now that we are open-source, we should have the link to this GitHub repository.

### Acceptance Criteria

We should display the GitHub logo just after our `Twitter` logo.

### Additional Context

Note that we use `react-feather`, which is an icon library where we get our social logos from.

### PR DESCRIPTION
**Pull request by vazquezea96**
## Description ✏️

Closes #37 

Describe what this PR does.

- Displays the GitHub Icon on the home page of the Member Profile that directs individuals to ColorStack's GitHub, as such:
image

## Type of Change 🐞

- [x] Feature - A non-breaking change which adds functionality.
- [ ] Fix - A non-breaking change which fixes an issue.
- [ ] Refactor - A change that neither fixes a bug nor adds a feature.
- [ ] Documentation - A change only to in-code or markdown documentation.
- [ ] Tests - A change that adds missing unit/integration tests.
- [ ] Chore - A change that is likely none of the above.

## Checklist ✅

- [x] I have done a self-review of my code.
- [ ] I have manually tested my code (if applicable).
- [ ] I have added/updated any relevant documentation (if applicable).

### CODE REVIEW

File: apps/member-profile/vite.config.ts.timestamp-1712340593047-e8a63840544c3.mjs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
...IHsgaW5zdGFsbEdsb2JhbHMgfSBmcm9tICdAcmVtaXgtcnVuL25vZGUnO1xuaW1wb3J0IHsgZGVmaW5lQ29uZmlnIH0gZnJvbSAndml0ZSc7XG5pbXBvcnQgdHNjb25maWdQYXRocyBmcm9tICd2aXRlLXRzY29uZmlnLXBhdGhzJztcblxuaW5zdGFsbEdsb2JhbHMoKTtcblxuZXhwb3J0IGRlZmF1bHQgZGVmaW5lQ29uZmlnKChlbnYpID0+IHtcbiAgcmV0dXJuIHtcbiAgICBwbHVnaW5zOiBbcmVtaXgoKSwgdHNjb25maWdQYXRocygpXSxcbiAgICBzZXJ2ZXI6IHsgcG9ydDogMzAwMCB9LFxuXG4gICAgLy8gVGhpcyBpcyBuZWVkZWQgdG8gc3VwcG9ydCB0aGUgdG9wLWxldmVsIGF3YWl0IGZvciBpbml0aWFsaXppbmcgZmVhdHVyZVxuICAgIC8vIGZsYWdzLiBXZSBvbmx5IGNoYW5nZSB0aGUgdGFyZ2V0IGZvciB0aGUgc2VydmVyIGJ1aWxkIHRob3VnaCwgdG8gYXZvaWRcbiAgICAvLyBhbnkgcG90ZW50aWFsIGlzc3VlcyB3aXRoIGJyb3dzZXIgY29tcGF0aWJpbGl0eS5cbiAgICAuLi4oZW52LmlzU3NyQnVpbGQgJiYgeyBidWlsZDogeyB0YXJnZXQ6ICdlc25leHQnIH0gfSksXG4gIH07XG59KTtcbiJdLAogICJtYXBwaW5ncyI6ICI7QUFBNFcsU0FBUyxjQUFjLGFBQWE7QUFDaFosU0FBUyxzQkFBc0I7QUFDL0IsU0FBUyxvQkFBb0I7QUFDN0IsT0FBTyxtQkFBbUI7QUFFMUIsZUFBZTtBQUVmLElBQU8sc0JBQVEsYUFBYSxDQUFDLFFBQVE7QUFDbkMsU0FBTztBQUFBLElBQ0wsU0FBUyxDQUFDLE1BQU0sR0FBRyxjQUFjLENBQUM7QUFBQSxJQUNsQyxRQUFRLEVBQUUsTUFBTSxJQUFLO0FBQUE7QUFBQTtBQUFBO0FBQUEsSUFLckIsR0FBSSxJQUFJLGNBQWMsRUFBRSxPQUFPLEVBQUUsUUFBUSxTQUFTLEVBQUU7QUFBQSxFQUN0RDtBQUNGLENBQUM7IiwKICAibmFtZXMiOiBbXQp9Cg==
```
Comment (tomas-salgado) [inline apps/member-profile/vite.config.ts.timestamp-1712340593047-e8a63840544c3.mjs:20]: I think you forgot to remove this file, it doesn't seem relevant to the icon changes

**Comment by vazquezea96** [comment]
@tomas-salgado removed the vite timestamp file that was added to the repo.

**Review by ramiAbdou — APPROVED** [review]
LGTM! 🚀
